android/src/main/java/com/swmansion/reanimated/NodesManager.java

```diff
@@ -109,13 +109,13 @@ public AnimationsManager getAnimationsManager() {
     return mAnimationManager;
   }
 
-  public void onCatalystInstanceDestroy() {
+  public void onInvalidate() {
```

Comment:
Shouldn't it be just `invalidate` instead of `onInvalidate` as in `ReanimatedModule`?